src/plugins/intel_cpu/thirdparty/CMakeLists.txt

```diff
@@ -154,7 +154,14 @@ function(ov_add_onednn)
     endif()
 endfunction()
 
+if(AARCH64 AND (NOT ANDROID))
+    set(ENABLE_SNIPPETS_LIBXSMM_TPP ON)
+endif()
+
```

Comment:
If we set this variable in `src/plugins/intel_cpu/CMakeLists.txt`, I think it should be visible in this file too and then no need to set one more time it here. What do you think?

Reply:
Yes, you are right. It is visible in thirdparty/CMakelists.txt. thanks Alexandra! removed the repeated set.